pkgs/by-name/ws/wsjtz/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  wsjtx,
+  fetchzip,
+  lib,
+}:
+
+wsjtx.overrideAttrs (old: rec {
+  pname = "wsjtz";
+  version = "2.7.0-rc7-1.43";
+
+  src = fetchzip {
+    url = "https://psychz.dl.sourceforge.net/project/wsjt-z/Source/${pname}-${version}.zip";
```

Comment:
Please don't use this.

Using `pname` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21

Reply:
Is it okay to use the version number? Or should that be hardcoded as well?